src/legacy/core_plugins/metrics/server/lib/vis_data/get_series_data.js

```diff
@@ -44,16 +46,21 @@ export async function getSeriesData(req, panel) {
   try {
     const data = await searchRequest.search({ body });
     const series = data.map(handleResponseBody(panel));
+    const lastSeriesTimestamp = getLastSeriesTimestamp(series);
```

Comment:
move this logic from `get_series_data.js`

Reply:
Moved